### PR TITLE
[stable/metabase] add persistent volume support

### PR DESCRIPTION
**Pull request by i1men**
support for persistent volumes

### CODE REVIEW

**Comment by zanhsieh** [comment]
@i1men 
Could you bump the chart version to latest and sign DCO please? Thank you.

**Comment by i1men** [comment]
DCO

File: stable/metabase/values.yaml [hunk 1]
```diff
@@ -76,6 +76,14 @@ ingress:
     #   hosts:
     #     - metabase.domain.com
 
+## Persist data to a persistent volume
+persistence:
+  enabled: false
+  ## Only when persistent enabled
+  # accessMode:
```
Comment (zanhsieh) [inline stable/metabase/values.yaml]: stable/metabase/values.yaml
  83:16     error    trailing spaces  (trailing-spaces)

reported by circle-ci lint

File: stable/metabase/values.yaml [hunk 2]
```diff
@@ -76,6 +76,14 @@ ingress:
     #   hosts:
     #     - metabase.domain.com
 
+## Persist data to a persistent volume
+persistence:
+  enabled: false
+  ## Only when persistent enabled
+  # accessMode: 
+  # size: 1Gi
+  # storageClass:
```
Comment (zanhsieh) [inline stable/metabase/values.yaml]: stable/metabase/values.yaml
  85:18     error    trailing spaces  (trailing-spaces)

reported by circle-ci lint

Comment (i1men) [reply stable/metabase/values.yaml]: thank you s much, i messed this

**Comment by maorfr** [comment]
ping @pmint93, mind reviewing this?
in addition, i have sent you an invitation to join the org as a trusted collaborator.

if you'd like to maintain this chart, would you mind submitting a PR adding an OWNERS file with you in it? assign that to me if you decide to do so.

thanks, and thanks @i1men for contributing!

**Comment by pmint93** [comment]
Using default embedded H2 database is not recommended for production use and scalable.
And, yet some block volume does not support ReadWriteMany, so we could end up at running single replicas metabase deployment here.
But, I can say this PR was fine for some use case.

**Comment by pmint93** [comment]
@maorfr Ok, let me create a PR to the OWNER file.
I should add my username in the `approvers` section, right ?

**Comment by SchroterQuentin** [comment]
Can someone approved this request ? I would love to use metabase with persistent volume.
Thanks in advance.

**Comment by pmint93** [comment]
@i1men Can you please bump chart to the latest version and resolve the current conflict ? then I can merge this for you.

**Comment by pmint93** [comment]
@i1men  Oops, the DCO check require all commits to be signed-off.
https://github.com/helm/charts/pull/17172/checks

**Comment by buzzedword** [comment]
@i1men would you mind if I take this back over the finish line? Unless you want to reopen it. It's such a great use case for testing in development clusters!

**Comment by i1men** [comment]
> @i1men would you mind if I take this back over the finish line? Unless you want to reopen it. It's such a great use case for testing in development clusters!

Sure ;)

**Comment by Uter1007** [comment]
is this already useable ? Did not find any settings in the documentation or template samples. 

Cheers,
Christoph

**Pull request closed without merging**